### PR TITLE
Feature/sr 10 implement hero section

### PR DESCRIPTION
**Pull request by GalacticOctopus**
Responsive Hero Page _(plus basic react router)_

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,33 +1,21 @@
...
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <Router>
+        <Routes>
+          <Route path="/home" element={<Home />}></Route>
+          <Route path="/" element={<Navigate to="/home" />}></Route>
```
Comment (masterHAWK99) [inline src/App.tsx:16]: Homepage should be at `/` path.

File: src/scss/pages/_home.scss [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  }
+
+  .text-body {
+    font-size: 1rem;
+  }
+
+  .btn {
+    flex-direction: row;
+  }
+}
```
Comment (masterHAWK99) [inline src/scss/pages/_home.scss:126]: Get rid of duplicated classess

File: src/scss/base/_typography.scss [hunk 3]
```diff
@@ -21,7 +21,9 @@
 .text-body {
   font-size: 1.25rem;
 }
-.text-body-sm {
+.text-body-sm,
+.text-body-sm-bold {
```
Comment (masterHAWK99) [inline src/scss/base/_typography.scss:25]: Create separated class called for example: `text-700` for this font weight.

File: src/pages/Hero.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        <div className="home__half">
+          <div className="home__wrapper2">
+            <img
+              src="src\assets\scooter1.png"
+              alt="Newest model electric scooter"
+            ></img>
+            <img
+              src="src\assets\scooter2.png"
+              alt="Person riding electric scooter on road"
+            ></img>
```
Comment (masterHAWK99) [inline src/pages/Hero.tsx:30]: Import those images

**Review by masterHAWK99 — CHANGES_REQUESTED** [review]

File: src/scss/pages/_home.scss [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 50%;
+    height: 100vh;
+    min-height: 1000px;
+  }
+
+  &__half:nth-child(2) {
+    background-color: #f7fcf5;
```
Comment (masterHAWK99) [inline src/scss/pages/_home.scss:20]: Use variables with colors

File: src/scss/pages/_home.scss [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+
+  &__wrapper2 {
+    display: flex;
+    align-items: flex-end;
+    background-color: #f7fcf5;
+  }
+
+  &__featured {
+    display: flex;
+    background-color: #f9f9f9;
```
Comment (masterHAWK99) [inline src/scss/pages/_home.scss:44]: Use variables with colors

**Review by masterHAWK99 — CHANGES_REQUESTED** [review]

File: src/pages/Hero.tsx (hunk 4 above)
Comment (GalacticOctopus) [reply src/pages/Hero.tsx:30]: You mean, by using props?

**Review by masterHAWK99 — APPROVED** [review]

**Pull request merged**